### PR TITLE
Disable whole module optimization and remove explicit Swift version from performance tests

### PR DESCRIPTION
**Pull request by cspickert**

### CODE REVIEW

File: BUILD [hunk 1]
```diff
@@ -154,7 +146,7 @@ swift_library(
     srcs = glob([
         "Tests/PromisesPerformanceTests/*.swift",
     ]),
-    copts = SWIFT_COPTS + [
+    copts = [
         "-swift-version",
         "4",
```
Comment (temrich) [inline BUILD]: @shoumikhin Do you know why this was pinned to Swift 4?

Comment (temrich) [reply BUILD]: @cspickert do you mind removing `-swift-version=4` for the perf tests target?

Comment (cspickert) [reply BUILD]: No problem, updated just now.

**Comment by temrich** [comment]
Thank you!

**Pull request merged**